src/app/app.component.ts

```diff
@@ -22,4 +22,8 @@ export class AppComponent {
   onSave(fhirResult): void {
     this.fhirPreview = JSON.stringify(fhirResult, null, 2);
   }
+
+  addTotalScoreRule(): void {
+    const result = this.ruleEditorService.addTotalScoreRule(fhir.phq9, context);
```

Comment:
Might be nice to add the result to `this.fhirPreview` so the user can see it.

Reply:
Would this line add it to the preview?

this.fhirPreview = JSON.stringify(result, null, 2);

Reply:
Yes

Reply:
I am hoping that change will allow the user see the variables and the calculatedExpression which were created by that button.